tools/SourceKit/tools/sourcekitd/include/sourcekitd/Internal.h

```diff
@@ -59,6 +59,7 @@ enum class CustomBufferKind {
   InheritedTypesArray,
   DocStructureElementArray,
   AttributesArray,
+  SyntaxTree
```

Comment:
This should be a more general buffer kind name, like `RawData`.
It could be used for other things, not just SyntaxTree.